source/agora/node/Ledger.d

```diff
@@ -931,7 +947,7 @@ unittest
 
             if (is_valid)
             {
-		ledger.forceCreateBlock();
```

Comment:
Oh I'm sorry for introducing tabs here.. It was my diff tool. I fixed it now. :)